src/components/select/select.component.ts

```diff
@@ -270,7 +272,7 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
             if (!this._element.nativeElement.contains(this._document.activeElement)) {
                 this.dropdownOpen = false;
                 if (!this.multiple) {
-                    this.input = this.getDisplay(this.value);
+                   this.input = this.getDisplay(this.value);

```

Comment:
This was an incorrect indentation change, please re-do indentation for the file just to make sure.